packages/fetchai/skills/confirmation_aw1/handlers.py

```diff
@@ -323,7 +325,9 @@ class LedgerApiHandler(Handler):
 
     def setup(self) -> None:
         """Implement the setup for the handler."""
-        pass
+        strategy = cast(Strategy, self.context.strategy)
+        for registered_aea in strategy.all_registered_aeas:
+            self._send_confirmation_details_to_aw2_aeas(registered_aea)
```

Comment:
On each restart we resend ALL registrations..responsibility of receiving aea to only save the new ones